Source/WebKit/_WebKit_SwiftUI/API/View+WebViewModifiers.swift

```diff
@@ -30,50 +30,56 @@ extension View {
     @available(WK_IOS_TBA, WK_MAC_TBA, WK_XROS_TBA, *)
     @available(watchOS, unavailable)
     @available(tvOS, unavailable)
-    public func webViewBackForwardNavigationGestures(_ value: WebView.BackForwardNavigationGesturesBehavior = .automatic) -> some View {
+    public nonisolated func webViewBackForwardNavigationGestures(_ value: WebView.BackForwardNavigationGesturesBehavior) -> some View {
         environment(\.webViewAllowsBackForwardNavigationGestures, value)
     }
 
-    @_spi(Private)
-    public func webViewMagnificationGestures(_ value: WebView.MagnificationGesturesBehavior) -> some View {
+    /// Determines whether magnify gestures change the view’s magnification.
```

Comment:
Nit: s/Determines/Specifies/, I think the wording here is just slightly reversed since to determine something would mean to figure it out (and probably return it to the caller).